modules/ags/src/main/scala/lucuma/ags/GuideStarCandidate.scala

```diff
@@ -103,10 +99,7 @@ object GuideStarCandidate {
           .map { case (b, v) => (b, v.value) }
 
         new GuideStarCandidate(
-          st.name.value match {
-            case GaiaNameRegex(d) => d.toLong
-            case _                => -1
-          },
+          GuideStarName.from(st.name.value).toOption.flatMap(_.toGaiaSourceId).getOrElse(-1),
```

Comment:
should this be an error instead?

Reply:
I don't know. I just kept the same behavior as before. I didn't know if it would break something somewhere (in explore) if I made it an error.